changelog/1.16.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "1.16.0"
+description: "Released on 02/22/2021"
+---
+
+### Breaking Changes ❗
+
+- The deprecated `Services` feature is now **removed**. All resources created
+  for Services are preserved for data recovery purposes (volumes, database
+  tables, etc.), but are not accessible through the Coder platform. Please
+  reference [Docker in Environments](../environments/cvms) for a guide on the
+  new workflow for running containers within your Environment.
+- The built-in Internal Extension Marketplace is now **removed**. All resources
+  created for this feature are preserved for data recovery purposes, but are not
+  accessible through the Coder platform. For air-gapped deployments, the
+  `Custom` Extension Marketplace configuration enables the use of
+  [Open VSX](https://github.com/eclipse/openvsx) as an alternative.
```

Comment:
We need to document how to recover extensions or migrate to OpenVSX.

Reply:
The provided text is sufficient.